web_src/js/asciinema/player.js

```diff
@@ -0,0 +1,10 @@
+import * as AsciinemaPlayer from 'asciinema-player';
```

Comment:
Don't unconditionally import as this will cause this JS to needlessly run on all pages.

Insted, use `await import` after checking existance of the class name and `return` from the init function if absent. Check other dynamically loaded JS like mermaid for an example on this.

Reply:
I've learned a lot. Could you please check if I did that in the right way?